api/ja/336_Register_ComplexTypeProperty.md

```diff
@@ -72,8 +72,8 @@ JSON
 |Name|ComplexTypeProperty名|桁数：1&#65374;128<br>文字種:半角英数字と-(半角ハイフン)と_(半角アンダーバー)<br>ただし、先頭文字に-(半角ハイフン)と_(半角アンダーバー)は指定不可|○||
 |_ComplexType.Name|紐付くComplexType名|桁数：1&#65374;128<br>文字種:半角英数字と-(半角ハイフン)と_(半角アンダーバー)<br>ただし、先頭文字に-(半角ハイフン)と_(半角アンダーバー)は指定不可|○||
 |Type|型定義|Edm.Boolean / Edm.String / Edm.Int32 / Edm.Single / Edm.Double / Edm.DateTime / 登録済みComplexType名|○||
-|Nullable|Null値許可|true / false<br>デフォルト値は Null|×||
-|DefaultValue|デフォルト値|下記表を参照<br>デフォルト値は Null|×||
+|Nullable|null値許可|true / false<br>デフォルト値は null|×||

```

Comment:
To avoid confusion, the Japanese should be like the following.  [Reference](https://www.ibm.com/support/knowledgecenter/ja/ssw_i5_54/sqlp/rbafynulls.htm)  

        ヌル値許可